### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-283.4]Text: Text truncated when text spacing tool is applied.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11385]
**Is this a bug, enhancement, or feature request?**
Bug

**Describe your proposal.**
Component having issue: https://sap.github.io/fundamental-ngx/#/core/wizard
Text truncated in wizard header when text spacing tool is applied
The different sections available in the wizard dialog is truncated on applying text spacing settings.

**Can you handle that on the application side**
No

**Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)**
All latest version

**If this is a bug, please provide steps for reproducing it; the exact components you are using;**

Go through video recording to reproduce exact issue 
https://teams.microsoft.com/l/message/19:52031686-a5f1-4208-991e-24053aee4bf4_972f5d73-1e4f-45d1-8d44-1fb46909ebda@unq.gbl.spaces/1707468258531?context=%7B%22contextType%22%3A%22chat%22%7D

**Please provide relevant source code (if applicable).**

**Please provide stackblitz example(s).**
In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes we have got this bug from Accessibility expert only.

**Did you check the documentation and the API?**
Yes

**Did you search for similar issues?**
**Is there anything else we should know?**

**Comment by droshev** [comment]
@JyotiKumari1507 We don't have access to the link. Can you create a stackblitz instead?

**Comment by JyotiKumari1507** [comment]
@droshev video https://sap-my.sharepoint.com/:v:/p/divya01/Ee8ya4dymiROlUmU34vmA1UBnvlRbZsYnQV5s8FuVEGN-Q?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D&e=q3Tceu should be accessible now

**Comment by JyotiKumari1507** [comment]
@droshev pls let us know update on this
cc: @divyamalhotra488

**Comment by mikerodonnell89** [comment]
Closing this, @JyotiKumari1507 if this issue persists, please open a new ticket with a detailed description of the problem and steps to reproduce the issue, or a stackblitz example

**Comment by mikerodonnell89** [comment]
Looks like a similar problem as [here](https://github.com/SAP/fundamental-ngx/issues/11297) and #11297 has a better description, so i'll reopen this
